qmat/qcoeff/__init__.py

```diff
@@ -29,13 +29,20 @@ def Q(self):
     def weights(self):
         raise NotImplementedError("mouahahah")
 
+    @property
+    def weightsSecondary(self):
+        """
+        These weights can be used to construct a secondary lower order method from the same stages.
+        """
+        raise NotImplementedError("Maybe the Merpeople on Europa know this.")
```

Comment:
You want me to put muahahahaha instead? :D

Reply:
Muahahahah will never be shown, as not overriding the properties returning this will raise an error during import (due to the check on registration). However, this property don't have to be overriden to register the $Q$-generators, and will be displayed when people use `genCoeffs(...,useEmbedded=True)` if the generator does not have embedded weights.

So simply use :
```python
@property
    def weightsEmbedded(self):
        raise NotImplementedError(f"no embedded weights implemented for {self.__class__.__name__}")
```